_build/resolvers/resolve.update.php

```diff
@@ -233,6 +233,11 @@ function cleanupPlugin($modx, $name)
                 addToSetting($modx, 'tinymcerte.plugins', 'autoresize');
             }
 
+            if ($oldPackage && $oldPackage->compareVersion('2.1.0-pl', '>')) {
+                addToSetting($modx, 'tinymcerte.plugins', 'modai');
+                addToSetting($modx, 'tinymcerte.toolbar1', '| modai_generate modai_generate_image modai_enhance');
```

Comment:
here should be a space before the `|`

Reply:
it's using implode with a space as a glue, so the space will get added by the implode